app/controllers/manage/finishers_controller.rb

```diff
@@ -21,9 +21,12 @@ def index
           first_finisher = Finisher.order(:joined_on).first
           first = first_finisher.joined_on&.beginning_of_month || first_finisher.created_at.beginning_of_month
           last = Time.zone.today.beginning_of_month
-          @months = (first..last).map { |date| date.strftime("%Y-%m-01") }.uniq.reverse
+          @months = (first.to_datetime..last.to_datetime).map { |date| date.strftime("%Y-%m-01") }.uniq.reverse
```

Comment:
I ran into `TypeError: can't iterate from ActiveSupport::TimeWithZone` during testing so I added this to be safe.